extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css

```diff
@@ -232,3 +252,57 @@ amp-story-shopping-attachment a {
 [dir=rtl] .i-amphtml-amp-story-shopping-pdp-carousel-card:last-child {
   border-radius: 1.1em 0 0 1.1em !important;
 }
+
+
+/* PDP deatils */
+
+.i-amphtml-amp-story-shopping-pdp-details {
+  padding: 0 var(--i-amphtml-shopping-attachment-horizontal-padding) !important;
+}
+
+.i-amphtml-amp-story-shopping-pdp-details-header {
+  cursor: pointer !important;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.i-amphtml-amp-story-shopping-pdp-details-header-text {
+  pointer-events: none;
+}
+
+.i-amphtml-amp-story-shopping-pdp-details-header-caret {
```

Comment:
Nit (optional): Think it's worth updating this to something like `i-amphtml-amp-story-shopping-pdp-details-header-dropdown-caret` for a little additional clarity? I only suggest this because I had to look at the image in the description in order to know what "caret" here was referring to